manual/source/Tutorial/DataStructures/Graphs.rst

```diff
@@ -53,7 +52,7 @@ Later, for computing the shortest path we will also need ``<seqan/graph_algorith
 We want to model the network of cities as an undirected graph and label the edges with distances.
 Before we start creating edges and vertices, we need some typedefs to specify the graph type.
 
-SeqAn offers different specializations of the class :dox:`Graph`:, :dox:`UndirectedGraph Undirected Graph`, :dox:`DirectedGraph`, :dox:`Tree`, :dox:`Automaton`, :dox:`HmmGraph`, and :dox:`AlignmentGraph Alignment Graph`.
+SeqAn offers different specializations of the class :dox:`Graph`: :dox:`UndirectedGraph Undirected Graph`, :dox:`DirectedGraph`, :dox:`Tree`, :dox:`WordGraph Word Graph`, :dox:`Automaton`, :dox:`HmmGraph`, and :dox:`AlignmentGraph Alignment Graph`.
```

Comment:
hm but the following are the subclasses of :dox:`Graph` aren't they? 
so the content tries to say..
different specializations of class A: subclass 1, subclass 2...
so I still think the ":" is thus correct? I don't see why you should use only "," ?